eZ/Publish/Core/settings/slot.yml

```
@@ -0,0 +1,8 @@
+services:
+
+    ezpublish.api.slot.user.delete_user_signal:
+        class: eZ\Publish\Core\Repository\Slot\User\DeleteUser
+        arguments:
+            - "@ezpublish.api.service.content_type"
```

```suggestion
            - '@ezpublish.api.service.content_type'
```
Using double quotes was my mistake from the past, now keeping them for consistency inside single file. For new ones single quotes should be used.